neo/rawio/baserawio.py

```diff
@@ -182,6 +184,15 @@ def __init__(self, use_cache: bool = False, cache_path: str = "same_as_resource"
         self.header = None
         self.is_header_parsed = False
 
+        self._has_buffer_description_api = False
+
+    def has_buffer_description_api(self):
```

Comment:
```suggestion
    def has_buffer_description_api(self) -> bool:
```
I know you don't like typing, but we make this mistake too often distguishing the function from the attribute. So I want this to say it returns bool so we remember it is the function. I hope this will prevent problems where just saying the function returns truthy.